### ISSUE TITLE
Damaged options Xsane

### ISSUE THREAD

**Issue by vchess** [utsushi/utsushi#12]
Hi.
I rebuilt driver and found that damage options xsane

![xsane](https://cloud.githubusercontent.com/assets/12548639/10818496/19224756-7e53-11e5-9043-26c74fea2361.png)

xsane - v.999
sane - 1.0.24
boost - 1.58.0 <--- I think that the problem in it
GraphicsMagick - 1.3.20

```
gcc version 5.2.1 20150716
./configure --with-jpeg --with-tiff --with-sane --with-magick --with-magick-pp  --without-gtkmm  --disable-static
```

```
LANG=C apt-cache depends imagescan

imagescan-3.13.0-alt1
  Depends: </etc/udev/rules.d>
    udev-rules-1:226-alt1
  Depends: </lib64/ld-linux-x86-64.so.2>
    glibc-core-6:2.17-alt8
  Depends: <libGraphicsMagick++.so.3()(64bit)>
    libGraphicsMagick-c++-1.3.20-alt1.1.1
  Depends: <libGraphicsMagick.so.3()(64bit)>
    libGraphicsMagick-1.3.20-alt1.1.1
  Depends: <libboost_filesystem.so.1.58.0()(64bit)>
    libboost_filesystem1.58.0-1:1.58.0-alt1
  Depends: <libboost_program_options.so.1.58.0()(64bit)>
    libboost_program_options1.58.0-1:1.58.0-alt1
  Depends: <libboost_regex.so.1.58.0()(64bit)>
    libboost_regex1.58.0-1:1.58.0-alt1
  Depends: <libboost_system.so.1.58.0()(64bit)>
    libboost_system1.58.0-1:1.58.0-alt1
  Depends: <libboost_thread.so.1.58.0()(64bit)>
    libboost_thread1.58.0-1:1.58.0-alt1
  Depends: <libc.so.6(GLIBC_2.14)(64bit)>
    glibc-core-6:2.17-alt8
  Depends: <libc.so.6(GLIBC_2.15)(64bit)>
    glibc-core-6:2.17-alt8
  Depends: <libc.so.6(GLIBC_2.2.5)(64bit)>
    glibc-core-6:2.17-alt8
  Depends: <libc.so.6(GLIBC_2.3.2)(64bit)>
    glibc-core-6:2.17-alt8
  Depends: <libc.so.6(GLIBC_2.3.4)(64bit)>
    glibc-core-6:2.17-alt8
  Depends: <libc.so.6(GLIBC_2.4)(64bit)>
    glibc-core-6:2.17-alt8
  Depends: <libgcc_s.so.1(GCC_3.0)(64bit)>
    libgcc1-5.2.1-alt1
  Depends: <libjpeg.so.62()(64bit)>
    libjpeg-2:1.3.1-alt0.1
  Depends: <libltdl.so.7()(64bit)>
    libltdl7-2.4.2-alt4
  Depends: <libm.so.6(GLIBC_2.2.5)(64bit)>
    glibc-core-6:2.17-alt8
  Depends: <libstdc++.so.6(CXXABI_1.3)(64bit)>
    libstdc++6-5.2.1-alt1
  Depends: <libstdc++.so.6(GLIBCXX_3.4)(64bit)>
    libstdc++6-5.2.1-alt1
  Depends: <libstdc++.so.6(GLIBCXX_3.4.11)(64bit)>
    libstdc++6-5.2.1-alt1
  Depends: <libstdc++.so.6(GLIBCXX_3.4.15)(64bit)>
    libstdc++6-5.2.1-alt1
  Depends: <libstdc++.so.6(GLIBCXX_3.4.20)(64bit)>
    libstdc++6-5.2.1-alt1
  Depends: <libstdc++.so.6(GLIBCXX_3.4.21)(64bit)>
    libstdc++6-5.2.1-alt1
  Depends: <libstdc++.so.6(GLIBCXX_3.4.9)(64bit)>
    libstdc++6-5.2.1-alt1
  Depends: <libtiff.so.5()(64bit)>
    libtiff5-4.0.3-alt1
  Depends: <libudev.so.1()(64bit)>
    libudev1-1:226-alt1
  Depends: <libudev.so.1(LIBUDEV_183)(64bit)>
    libudev1-1:226-alt1
  Depends: <libusb-1.0.so.0()(64bit)>no effect
    libusb-1.0.20-alt1
  Depends: <rtld(GNU_HASH)>
    glibc-core-6:2.17-alt8
  Depends: <rtld(GNU_UNIQUE)>
    glibc-core-6:2.17-alt8
```

Run LANG=POSIX xsane no effect.

**Comment by paddy-hack** [comment]
This is due to a bug in the `utsushi` SANE backend.  It is triggered by changes in the `std::string` implementation of `libstdc++`.  I'm waiting for an upstream release that fixes this.

**Comment by paddy-hack** [comment]
I just pushed updates that should fix this issue.  If you could test, that would be appreciated.

For `utsushi` the fix is in utsushi/utsushi@0c32dad.  If you prefer one of the `imagescan` branches instead, use utsushi/imagescan@f333e9d (`vcfiles` branch) or utsushi/imagescan@ec41a6f (`archive` branch).
